torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -1658,6 +1658,11 @@ void LoopNest::unroll(For* f, Stmt** unrolled) {
   p->replace_stmt(f, *unrolled);
 }
 
+void LoopNest::unroll(For* f) {
+  Stmt* unrolled;
```

Comment:
Can you initialize this to `nullptr` so that clang-tidy doesn't complain?

Reply:
Oh, right. For consistency, I copied what `sliceHead` does and added an annotation to prevent the complains :)